src/inc/dacprivate.h

```diff
@@ -20,6 +20,14 @@
 #include <msodw.h>
 #endif // FEATURE_PAL
 
+#if !defined(MSLAYOUT)
```

Comment:
Btw, if you do this, could you please copy the code from the common.h as is, including the comment? Also, you don't need to add extra check for __clang__, the _GNU_C_ is good enough (defined for clang too).